packaging/README.md

```diff
@@ -0,0 +1,33 @@
+# Distribute and Install nmstate
+The packaging and distribution of nmstate is currently implemented through
```

Comment:
IMHO the sentence is more complicated that necessary, it could just be:

`NMState is distributed on PyPI and Docker Hub.`

Also it would be great to agree on a capitalization for NMState. The main web page calls it NMState and not nmstate.

Reply:
Done